### PR TITLE
add calc to targets.json

### PR DESCRIPTION
**Pull request by jseppi**
Adds CALC

### CODE REVIEW

File: config/targets.json [hunk 1]
```diff
@@ -203,5 +203,19 @@
...
+  {
+    "name": "calc",
+    "slack_channel": "calc-data-capture",
+    "links": [
+      {
+        "url": "https://calc-dev.apps.cloud.gov/",
+        "text": "CALC (staging)"
+      },
+      {
+        "url": "https://calc.gsa.gov/",
```
Comment (jseppi) [inline config/targets.json]: As noted in #103, perhaps this should not point to the production URL. Let me know if I should remove that entry from `links`.

Comment (afeld) [inline config/targets.json]: Yeah, no need for both. I'll take care of it.

**Comment by afeld** [comment]
Results are up! https://compliance-viewer.18f.gov/results/calc/current

**Comment by jseppi** [comment]
Thanks!

**Pull request merged**